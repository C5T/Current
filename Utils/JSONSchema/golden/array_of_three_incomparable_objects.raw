{"Array":{"element":{"Object":{"fields":{"z":{"String":{"values":{"c":1},"counters":[[1,"c"]],"instances":1,"nulls":1}},"y":{"String":{"values":{"b":1},"counters":[[1,"b"]],"instances":1,"nulls":2}},"x":{"String":{"values":{"a":1},"counters":[[1,"a"]],"instances":1,"nulls":2}},"p":{"String":{"values":{"q":3},"counters":[[3,"q"]],"instances":3,"nulls":0}}},"instances":3,"nulls":0}},"instances":1,"nulls":0}}