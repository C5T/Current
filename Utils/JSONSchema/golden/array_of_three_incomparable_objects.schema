// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Element_Object) {
  CURRENT_FIELD(x, std::string);
  CURRENT_FIELD(p, std::string);
  CURRENT_FIELD(y, std::string);
  CURRENT_FIELD(z, std::string);
};

using Schema = std::vector<Schema_Element_Object>;
