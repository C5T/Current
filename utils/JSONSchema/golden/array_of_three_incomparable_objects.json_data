[ {"x":"a","p":"q"}, {"y":"b","p":"q"}, {"z":"c","p":"q"} ]
