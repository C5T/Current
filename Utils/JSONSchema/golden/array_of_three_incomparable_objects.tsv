Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	3	0	4 fields	p, x, y, z
Schema[].p	String	3	0	1 distinct value, "q":3
Schema[].x	String	1	2	1 distinct value, "a":1
Schema[].y	String	1	2	1 distinct value, "b":1
Schema[].z	String	1	1	1 distinct value, "c":1
